lib/ansible/modules/cloud/amazon/redshift.py

```diff
@@ -132,6 +132,18 @@
     description:
       - how long before wait gives up, in seconds
     default: 300
+  tags:
+    description:
+      - A dictionary of resource tags.
+    type: dict
+    aliases: ['resource_tags']
+    version_added: 2.9
```

Comment:
```suggestion
    version_added: 2.10
```
2.9 was frozen and branched this week.